PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
```

Comment:
There's a missing space here. There should always be two blank spaces above function names unless they're nested within a class or function, in which case there should be one blank space.